schema-engine/connectors/mongodb-schema-connector/tests/introspection/types/mod.rs

```diff
@@ -442,13 +441,15 @@ fn empty_arrays() {
     "#]];
 
     expected.assert_eq(res.datamodel());
-    res.assert_warning_code(103);
-    res.assert_warning("Could not determine the types for the following fields.");
 
-    res.assert_warning_affected(&json!([{
-        "model": "A",
-        "field": "data",
-    }]));
+    let expect = expect![[r#"
+        *** WARNING ***
+
+        Could not determine the types for the following fields:
+          - model: A, field: data
```

Comment:
Also nitpicking here; are these things we want to capitalise in our output? `Model: ...` vs `model:  ...`. I think it'd look better capitalised 😅